pyanaconda/storage: update fstab by appending

The 'anaconda.storage.write' has updated fstab by overwriting
the original one, that cause the mountall.sh could not work
correctly at system boot time, we should update fstab by
appending to the original could fix this issue.

Upstream-Status: Inappropriate [WR specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/storage/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/storage/__init__.py b/pyanaconda/storage/__init__.py
--- a/pyanaconda/storage/__init__.py
+++ b/pyanaconda/storage/__init__.py
@@ -2369,7 +2369,7 @@ class FSSet(object):
             fstab += format % ('tmpfs', '/var/volatile', 'tmpfs',
                                'defaults', 0, 0)
 
-        open(fstab_path, "w").write(fstab)
+        open(fstab_path, "a").write(fstab)
 
         # /etc/crypttab
         crypttab_path = os.path.normpath("%s/etc/crypttab" % instPath)
-- 
1.8.1.2

